3_javascript/javascript1/week4/Homework/script.js

```diff
@@ -0,0 +1,68 @@
+let userName = "";
+const todoList = [];
+let timerId;
+
+function getReply(command) {
+  if (command.toLowerCase().includes("hello my name is")) {
```

Comment:
In order to reduce duplicated code, you can either store this in a separate variable like this
```
const lcCommand = command.toLowerCase()
 if (lcCommand.includes("hello my name is")) {
```
or create some helper function :)